packages/react-hot-loader/README.md

```diff
@@ -49,8 +49,9 @@ module.exports = {
 
 > Note: Make sure to set the `output.publicPath` property to `"/"` as well. Otherwise hot reloading won't work as expected for nested routes.
 
-4. Wrap your application into `<AppContainer>`, all children of `<AppContainer>` will be reloaded when a change occurs:
+4. Setup React-hot-loader to update your application when its code updates.
 
+4.1 Wrap your application into `<AppContainer>`, all children of `<AppContainer>` will be reloaded when a change occurs:
```

Comment:
This part should not appear in this section. It is an advanced case. We have to keep it as simple as possible for the user and the `hotExported` is easier.